README.rst

```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
 .. image:: http://img.shields.io/:license-Apache%202-brightgreen.svg
     :target: http://www.apache.org/licenses/LICENSE-2.0.txt
 
-*Private Preview. Linux Only. No PHP 5 support. PHP 7.2 only.*
+Snowflake provides a driver that uses the 
+`PHP Data Objects (PDO) extension <https://www.php.net/manual/en/book.pdo.php>`_.
+to connect to the Snowflake database.
 
-Configuring Environment
+.. contents::
+
+Prerequisites
 ================================================================================
 
-PHP Versions and Extensions
-----------------------------------------------------------------------
+In order to build the Snowflake PHP PDO Driver, you must have the following software installed:
 
-PHP 7.2 is supported. The following extensions are required:
+- (Windows) Visual Studio
+- 
```

Comment:
What needs to be installed for Linux and macOS?

Reply:
For Linux you need `gcc` (version 5.2 or higher) and for macOS you need `clang` (whatever version can be installed on a user's macOS version)

Reply:
jq is needed on all platforms to run test script. Nothing special besides that, the default compiler on Linux and OSX work well.

Reply:
Thanks! I updated the docs.